src/main.rs

```diff
@@ -16,56 +17,99 @@ use de_gui::GuiPluginGroup;
 use de_index::IndexPluginGroup;
 use de_loader::LoaderPluginGroup;
 use de_lobby_client::LobbyClientPluginGroup;
+use de_log::LogPluginGroup;
 use de_menu::MenuPluginGroup;
 use de_movement::MovementPluginGroup;
 use de_objects::ObjectsPluginGroup;
 use de_pathing::PathingPluginGroup;
 use de_signs::SignsPluginGroup;
 use de_spawner::SpawnerPluginGroup;
 use de_terrain::TerrainPluginGroup;
+use tracing::{span, Level};
+use tracing_subscriber::layer::SubscriberExt;
+use tracing_subscriber::{fmt, EnvFilter};
 
 const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 const GIT_SHA: &str = env!("GIT_SHA");
 
 fn main() {
-    let mut app = App::new();
-    app.insert_resource(Msaa::Sample4)
-        .add_plugins(DefaultPlugins.set(WindowPlugin {
-            primary_window: Some(Window {
-                title: "Digital Extinction".to_string(),
-                mode: WindowMode::BorderlessFullscreen,
-                ..Default::default()
-            }),
-            ..default()
-        }))
-        .add_plugin(LogDiagnosticsPlugin::default())
-        .add_plugin(FrameTimeDiagnosticsPlugin::default())
-        .add_plugin(GamePlugin)
-        .add_plugins(ConfigPluginGroup)
-        .add_plugins(GuiPluginGroup)
-        .add_plugins(LobbyClientPluginGroup)
-        .add_plugins(MenuPluginGroup)
-        .add_plugins(CorePluginGroup)
-        .add_plugins(ObjectsPluginGroup)
-        .add_plugins(TerrainPluginGroup)
-        .add_plugins(LoaderPluginGroup)
-        .add_plugins(IndexPluginGroup)
-        .add_plugins(PathingPluginGroup)
-        .add_plugins(SignsPluginGroup)
-        .add_plugins(SpawnerPluginGroup)
-        .add_plugins(MovementPluginGroup)
-        .add_plugins(ControllerPluginGroup)
-        .add_plugins(CameraPluginGroup)
-        .add_plugins(BehaviourPluginGroup)
-        .add_plugins(CombatPluginGroup)
-        .add_plugins(ConstructionPluginGroup);
+    // for file name
+    let dt = chrono::Local::now();
```

Comment:
I would move all this logic to `de_log`.

Note that the log file is passed to the logging plugin in an unnecessary complex manner (public property of the plugin group, then the plugin then a resource). None of that would be needed if the setup is done in a startup plugin.